ideascube/utils.py

```diff
@@ -1,8 +1,27 @@
+from io import TextIOWrapper
 import sys
 
 from django.conf import locale
 
 
+class AutomagicTextIOWrapper(TextIOWrapper):
```

Comment:
> This is why I suggest to just put it on the class itself: curious people will end there if they want to understand why we use it :)

We use it for the exact same reasons we'd use a `TextIOWrapper`. Also, it's not a very specific class, and (if moved to use cchardet) could even be sent to upstream cchardet.

In that scenario, where it gets accepted in cchardet and we use it from them, where would you put the comment?

> About the name: not sure we need to keep the reference to TextIoWrapper in it.

It really is a TextIOWrapper though, with all its bells and whistles. We use it only to decode as we iterate, but we could use it for something else. (e.g call its `.read()` method)

In fact, I'm tempted to name it just `TextIOWrapper`, since we'd just use it everywhere in Ideascube instead of Python's `TextIOWrapper`. :smile: